netbox/vpn/choices.py

```diff
@@ -255,6 +256,9 @@ class L2VPNTypeChoices(ChoiceSet):
             (TYPE_EPTREE, _('Ethernet Private Tree')),
             (TYPE_EVPTREE, _('Ethernet Virtual Private Tree')),
         )),
+        ('SPB', (
```

Comment:
I think it makes more sense to just define this as an "Other" section.  This will allow us to expand with other protocols if needed.

If we decide to add additional SPB techniques later, we can then add a SPB category and move them around (the categories have no bearing on the actual functionality of the model, they are just categories)

```suggestion
        ('Other', (
```